projects/storefrontlib/src/lib/store-finder/store/reducers/find-stores.reducer.ts

```diff
@@ -16,58 +16,24 @@ export function reducer(
 ): FindStoresState {
   switch (action.type) {
     case fromStoreFinder.ON_HOLD:
-    case fromStoreFinder.FIND_STORES: {
-      return {
-        ...state,
-        isLoading: true
-      };
-    }
-
-    case fromStoreFinder.FIND_STORES_SUCCESS: {
-      const findStoresEntities = action.payload;
-
-      return {
-        ...state,
-        findStoresEntities,
-        isLoading: false
-      };
-    }
-
-    case fromStoreFinder.FIND_STORES_FAIL: {
-      return {
-        ...state,
-        isLoading: false
-      };
-    }
-
-    case fromStoreFinder.FIND_ALL_STORES_BY_COUNTRY_SUCCESS: {
-      const findStoresEntities = action.payload;
-
-      return {
-        ...state,
-        findStoresEntities
-      };
-    }
-
-    case fromStoreFinder.FIND_ALL_STORES_BY_COUNTRY_FAIL: {
-      return {
-        ...state
-      };
+    case fromStoreFinder.FIND_STORES:
+    case fromStoreFinder.FIND_ALL_STORES_BY_COUNTRY:
+    case fromStoreFinder.FIND_ALL_STORES_BY_REGION: {
+      return { ...state, isLoading: true };
```

Comment:
This is technically ok but it's still better to be more verbose and cover individual cases. What if one of the actions has to change behavior, then we still have to open these up. It's not a bad thing having a few extra lines of code

Reply:
lint/sonar will complain if you have blocks with same body